plugins/modules/system/xfconf.py

```diff
@@ -144,12 +145,11 @@ def __init__(self, module):
         self.method_map = dict(zip((self.SET, self.GET, self.RESET),
                                    (self.set, self.get, self.reset)))
 
-        # @TODO This will not work with non-English translations, but xfconf-query does not return
-        #       distinct result codes for distinct outcomes.
         self.does_not = 'Property "{0}" does not exist on channel "{1}".'.format(self.property, self.channel)
 
         def run(cmd):
-            return module.run_command(cmd, check_rc=False)
+            # Force language to 'C' to ensure return values are always formatted in english, even in non-english environments
+            return module.run_command(cmd, check_rc=False, environ_update=dict(LANG='C'))
```

Comment:
Instead of passing `environ_update` here, set `module.run_command_environ_update = dict(LANG='C')` at the beginning of the module code instead. Then this will be passed for all `module.run_command` calls.